### ISSUE TITLE
Feature Request: notifications link destinations

### ISSUE THREAD

**Issue by JerryatricBird** [bitwave-tv/bitwave#180]
When looking at my notifications, a lot of the time the stream is already over. To that end, it would be great if clicking on the notification linked directly to the particular replay that matches the stream that triggered the notification. If the stream is still live, the link would obviously still just link to the channel.

**Comment by diingus** [comment]
What if the stream wasn't archived? Would the channel page be an appropriate destination link then? Maybe with a toast that explained that 'this stream is not in the archive'.

**Comment by JerryatricBird** [comment]
yes. Just do a check if a replay exists. if so direct to there. if not, send to channel. Another in between scenario is that the replay isn't loaded yet. This would probably be very common. Honestly, the replay page should show up right away after a stream ends, but with the message (replay is processing... please come back in (ETA)" so that the notifications in this scenario can link to them. Also it is helpful to get confirmation that a replay is on its way, even if it hasn't loaded yet.

**Comment by JerryatricBird** [comment]
by "the replay page should show up right away aft a stream ends" I mean that a replay page should be created right away. When a stream ends, it should play bumps as usual for those already watching the channel.

**Comment by diingus** [comment]
> Honestly, the replay page should show up right away after a stream ends, but with the message (replay is processing... please come back in (ETA)"

Definitely a quality-of-life feature I'd expect to be implemented. Could you open a separate issue for this? Thanks.
